### ISSUE TITLE
Menu autoclosing when used in an appbar component

### ISSUE THREAD

**Issue by dtomlinson91** [Alexeykhr/vue-stripe-menu#133]
Thanks for the work you've put into this - it's really good!

I'm having trouble when using this in an appbar component. It seems if I place the stripe-menu in an appbar component, whenever I move the mouse off the appbar component the focus is lost and the menu closes.

A quick video should (hopefully) explain what I mean:

https://imgur.com/J4Qdioq

My App.vue looks like:

```
<template>
  <div>
    <v-app :style="{ background: $vuetify.theme.themes.light.background }">
      <v-content>
        <Appbar></Appbar>
```

And it is in this Appbar component I'm placing the stripe-menu:

```
<template>
  <div>
    <v-app-bar
      flat
      app
      hide-on-scroll
      color="rgba(93.3%, 93.3%, 93.3%, 0.5)"
      v-if="!this.$store.getters.fullScreen"
    >
      <v-row>
 ...
        <v-col cols="4" class="d-flex justify-end">
          <vsm-menu
            ref="header"
            :menu="menu"
            :screen-offset="25"
            :base-width="380"
            :base-height="400"
          >
            <template #default="data">
              <component :is="data.item.content" class="content" />
              <component :is="data.item.secondary" class="content--secondary" />
            </template>
          </vsm-menu>
        </v-col>
        <v-spacer></v-spacer>
```

I'm using Vuetify, so the appbar has a default height of 64px. If my mouse scrolls outside of this appbar, the menu loses focus.

Is there and quick workaround or have I missed something obvious? I'm not sure if it's related to Vuetify (it could very well be but I have no idea where to begin looking it's not the nicest to dig into).

It's not the `<v-app-bar>` component - I've tried without it. It seems it loses focus if the mouse goes outside of the component that contains the strip-menu.

**Comment by Alexeykhr** [comment]
Hi, try this https://github.com/Alexeykhr/vue-stripe-menu/issues/109

**Comment by dtomlinson91** [comment]
@Alexeykhr thanks a lot mate, that did the trick.

The solution **was not** applying `z-index: 15;` to the `.vsm-menu` class, but to add it to the top-most parent element. In this case it was `<v-app-bar`>:

```
    <v-app-bar
      flat
      app
      hide-on-scroll
      color="rgba(93.3%, 93.3%, 93.3%, 0.5)"
      v-if="!this.$store.getters.fullScreen"
      style="z-index: 15;"
    >
```

https://imgur.com/a/OEYC3WU

This also fixes #109 (the image in the above video has position: absolute;)
